src/components/skeleton/Drawer.vue

```diff
@@ -0,0 +1,187 @@
+      </v-list-item>
+
+      <v-list-item link :href="analogistURL" ripple>
+        <img
+          style="max-width: 30px"
+          :src="require('../../../public/images/analogist-logo.png')"
+          class="mr-6 logo-white"
+          alt="analogist-img"
+        />
+
+        <v-list-item-content>
+          <v-list-item-title> Analogist </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+      <v-list-item link :href="ezpaarseDemoURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-chevron-double-right</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title v-text="$t('appbar.demonstration')"> </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+      <v-list-item link :href="ezmesureURL" ripple>
+        <img
+          style="max-width: 32px"
+          :src="require('../../../public/images/ezmesure-logo.svg')"
+          class="mr-6 logo-white"
+          alt="ezmesure-img"
+        />
+
+        <v-list-item-content>
+          <v-list-item-title> Ezmesure </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+      <v-list-item link :href="bibliomapURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-web</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> bibliomap </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+      <v-list-item link :href="ezpaarseDocURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-file-document-multiple</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> documentation </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="mailURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-email</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Mail </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="githubURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-github</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Github </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="twitterURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-twitter</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Twitter </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+
+       <v-list-item link :href="youtubeURL" ripple>
+        <v-list-item-icon>
+          <v-icon>mdi-youtube</v-icon>
+        </v-list-item-icon>
+
+        <v-list-item-content>
+          <v-list-item-title> Youtube </v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
```

Comment:
An array with `v-for` will be more readable (like in `ApplicationsMenu.vue` btw)